utils/scapp.h

```diff
@@ -429,14 +537,19 @@ scApp::scApp(string& version, string& defaultVersion,
       { "no_endpoint_rdo", argparser::option::no_argument, NULL, 1001 },
       { "no_selector_rdo", argparser::option::no_argument, NULL, 1002 },
       { "no_sse", argparser::option::no_argument, NULL, 1011 },
-      { "uastc", argparser::option::optional_argument, NULL, 1003 },
+	  { "uastc_level", argparser::option::required_argument, NULL, 1003 },
       { "uastc_rdo_l", argparser::option::optional_argument, NULL, 1004 },
       { "uastc_rdo_d", argparser::option::required_argument, NULL, 1005 },
       { "uastc_rdo_b", argparser::option::optional_argument, NULL, 1006 },
       { "uastc_rdo_s", argparser::option::optional_argument, NULL, 1007 },
       { "uastc_rdo_f", argparser::option::no_argument, NULL, 1008 },
       { "uastc_rdo_m", argparser::option::no_argument, NULL, 1009 },
-      { "verbose", argparser::option::no_argument, NULL, 1010 }
+	  { "verbose", argparser::option::no_argument, NULL, 1010 },
+	  { "astc_blk_s", argparser::option::required_argument, NULL, 1012 },
+	  { "astc_func", argparser::option::required_argument, NULL, 1013 },
+	  { "astc_mode", argparser::option::required_argument, NULL, 1014 },
+	  { "astc_quality", argparser::option::required_argument, NULL, 1015 },
+	  { "astc_normal", argparser::option::no_argument, NULL, 1016 }
```

Comment:
The indentation is 2 spaces too much on this and many of the previous lines.

Reply:
You should look at the last commit all files changed. These are fixed.